src/main/java/com/smartbear/collaborator/issue/IssueRest.java

```diff
@@ -739,10 +762,10 @@ public String createReview(CollabUserInfo collabUserInfo) throws Exception {
 			createReviewCommand.getArgs().put("title", issue.getSummary() != null ? issue.getSummary() : "");
 			createReviewCommand.getArgs().put("customFields",
 					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) )	});
-			createReviewCommand.getArgs().put("internalCustomFields",
-					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("JiraIssueId",  new HashSet(Arrays.asList(issue.getId().toString())) )});
+							new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) ),

```

Comment:
Sorry, but this is bad idea.
1. Collaborator server uses internal custom field "JiraIssueId" for updating review information on JIRA side. So internal custom field "JiraIssueId"  exists by default on Collab side
2. By default custom field "JiraIssueId" is not exists in review template, so if you don't create it manually on Collab side - you will get error while creating review.